src/openrct2/object/ObjectFactory.cpp

```diff
@@ -442,6 +442,27 @@ namespace ObjectFactory
                 {
                     throw std::runtime_error("Object has errors");
                 }
+                auto authors = json_object_get(jRoot, "authors");
+                if (authors != nullptr)
+                {
+                    if (json_is_array(authors))
```

Comment:
You can move this this check to up to ```if (authors != nullptr)```, json_is_array will evaluate to false if ```authors == nullptr```, the nullptr test is redundant this way.